solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1296,6 +1280,21 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         return _total.mul(_loss.d.sub(_loss.n)).div(_loss.d).add(_amount.mul(compN).div(compD));
     }
 
+    function getNetworkCompensation(uint256 _targetAmount, uint256 _baseAmount, PackedRates memory _packedRates) internal view returns (uint256) {
+        if (_targetAmount > _baseAmount) {
```

Comment:
@yudilevi : But it is much cleaner this way, because the reader can clearly see both expression within the same scope:
1. `_targetAmount > _baseAmount`
2. `_targetAmount - _baseAmount`

Reply:
more indentations are never cleaner, return statements mean the reader can ignore the entire code above the return

Reply:
Fixed.